src/platform/packages/shared/kbn-unified-data-table/src/components/compare_documents/hooks/calculate_diff.ts

```diff
@@ -38,11 +38,11 @@ export const calculateDiff = ({ diffMode, baseValue, comparisonValue }: Calculat
   }
 
   if (diffMode === 'words') {
-    return diffWords(formattedBaseValue, formattedComparisonValue, { ignoreWhitespace: false });
+    return diffWordsWithSpace(formattedBaseValue, formattedComparisonValue);
```

Comment:
`diffWordsWithSpace` does the same as `diffWords` with `ignoreWhitespace: false`, but they didn't include the latter in the TS definitions since they want consumers to just call `diffWordsWithSpace` instead.